engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/error/NoWrap.java

```diff
@@ -0,0 +1,9 @@
+package org.enso.interpreter.node.expression.builtin.error;
+
+import org.enso.interpreter.dsl.BuiltinType;
+import org.enso.interpreter.node.expression.builtin.Builtin;
+
+@BuiltinType(name = "Standard.Base.Data.Vector.No_Wrap")
```

Comment:
Off-topic, but I'd like to know the answer: What is `name` attribute **optional**? It is specified on some of the builtins, but not on all. Shouldn't it be specified all the time? I am asking because of

- #8852

the proposed `Standard.Prelude` should include all the builtins and their methods, but we want to have their names exactly the same as in `Standard.Base` - can we generate such a `Prelude.enso` from these annotations?